.github/workflows-source/release-testing.yml

```diff
@@ -46,44 +46,29 @@ jobs:
     uses: ./.github/workflows/ci-main.yml
     secrets: inherit
 
-  bazel-system-test-nightly:
-    name: Bazel System Test Nightly
+  # Run some extra system tests that are skipped on CI Main.
+  # Those test suites would overwhelm a runner, so each is run
+  # as a separate job.
+  bazel-system-tests:
```

Comment:
The ID `bazel-system-tests` is quite generic. I would suggest renaming this to `release-system-tests` or something similar with the name "release" in it. That way it's easy to see when Release Testing is being run when looking at the [Farm Dashboard](https://grafana.ch1-obsstage1.dfinity.network/d/uwEFG_yGk/farm-dashboard?orgId=1&from=now-24h&to=now&refresh=10s&timezone=utc&var-datasource=P36978C0E868FFF7F).